arm-containerregistry/2017-06-01-preview/swagger/containerregistry.json

```diff
@@ -121,8 +121,8 @@
         "tags": [
           "Registries"
         ],
-        "description": "Creates a container registry with the specified parameters.",
-        "operationId": "Registries_Create",
+        "description": "Creates or updates a container registry with the specified parameters.",
+        "operationId": "Registries_CreateOrUpdate",
```

Comment:
I had the same feeling and asked this question earlier. Yes, update can update more than just tags (actually, just one more -- `adminUserEnabled`). Does ARM allow removing an operation? We are okay either way at this point.